guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -219,6 +219,161 @@ Create the host from an image with the `hammer host create` command and include
+* VMware vCenter applies customization for the virtual machine's identity, including the host name, IP, and DNS
+* The virtual machine builds, `cloud-init` is invoked and pings {Project} on port `80`, which then redirects to `443`
+
+.Port and Firewall Requirements
+
+Because of the `cloud-init` service, the virtual machine always calls back to {Project} even if you register the virtual machine to {SmartProxy}. Ensure that you configure port and firewall settings to open any necessary connections.
+
+ifeval::["{build}" == "satellite"]
+For more information about port and firewall requirements, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.4/html/installing_satellite_server_from_a_connected_network/preparing_your_environment_for_installation#ports_prerequisites[Port and Firewall Requirements] in the _Installation Guide_.
+endif::[]
+
+.Associating the `userdata` and `Cloud-init` Templates with the Operating System
+
+. In the {Project} web UI, navigate to *Hosts* > *Operating Systems*, and select the operating system that you want to use for provisioning.
+. Click the *Template* tab.
+. From the *Cloud-init template* list, select *Cloudinit default*.
+. From the *User data template* list, select *UserData open-vm-tools*.
+. Click *Submit* to save the changes.
+
+.Preparing an Image to use the cloud-init Template
+
+To prepare an image, you must first configure the settings that you require on a virtual machine that you can then save as an image to use in {Project}.
+
+To use the `cloud-init` template for provisioning, you must configure a virtual machine so that `cloud-init` is installed, enabled, and configured to call back to {ProjectServer}.
+
+ifeval::["{build}" == "satellite"]
+For security purposes, you must add a Satellite X509 certificate to use HTTPs for all communication. This procedure includes steps to clean the virtual machine so that no unwanted information transfers to the image you use for provisioning.
+endif::[]
+
+If you have an image with `cloud-init`, you must still follow this procedure to enable `cloud-init` to communicate with {Project} because `cloud-init` is disabled by default.
+
+. On the virtual machine that you use to create the image, install `cloud-init`, `open-vm-tools`, and `perl`:
++
+----
+# yum -y install cloud-init open-vm-tools perl
+----
++
+. Create a configuration file for `cloud-init`:
++
+[options="nowrap" subs="+quotes"]
+----
+# vi /etc/cloud/cloud.cfg.d/_example_cloud-init_config_.cfg
+----
++
+. Add the following information to the `_example_cloud_init_config_.cfg` file:
++
+----
+datasource_list: [NoCloud]
+datasource:
+  NoCloud:
+    seedfrom: https://{foreman.example.com}/userdata/
+EOF
+----
++
+. Enable the CA certificates for the image:
++
+----
+# update-ca-trust enable
+----
++
+. Download the `katello-server-ca.crt` file from {ProjectServer}:
++
+[options="nowrap" subs="+quotes"]
+----
+# wget http://_{foreman.example.com}_/pub/katello-server-ca.crt
```

Comment:
Probably this is better:

    # wget -O /etc/pki/ca-trust/source/anchors/cloud-init-ca.crt http://_{foreman.example.com}_/pub/katello-server-ca.crt

The `mv` command below can be skipped then. But here is one thing - this will only work for katello deployment (and satellite). Let's add a note for "upstream" build: Server certificate is published only for Katello deployment, for Foreman deployment the server certificate must be copied from Apache configuration.